models/cam-fv/readme.rst

```diff
@@ -118,8 +123,9 @@ Reanalyses
 There have been two large-scale reanalysis efforts using CAM-FV and DART. 
 The **CAM6 Data Assimilation Research Testbed (DART) Reanalysis**
 is archived in the NSF NCAR Research Data Archive 
-`DS345.0 <https://rda.ucar.edu/datasets/ds345.0/#!description>`__ .
-(See the |CAM6_Rean|_ ).
+`d345000 <CAM6rean_rda_>`_ .
+(See the `CAM6rean_wiki`_ for details).
```

Comment:
You've changed the name of the link here.  Main branch:
(See the 1 degree reanalysis wiki ).

This pull request:
(See the CAM6rean_wiki for details).

Reply:
That was a reference formatting error.  But the doc still refers to the wiki by different words.  Is that a problem if the rst name of the link is consistent?

Reply:
no not a problem just checking that you were ok with having the link test be CAM6rean_wiki vs 1 degree reanalysis wiki. Either seems ok to me